UnityProject/Assets/Scripts/Core/UniversalObjectPhysics.cs

```diff
@@ -530,12 +531,11 @@ public void StoreTo(ObjectContainer newParent)
 		if (newParent.OrNull()?.gameObject == this.gameObject)
 		{
 			Chat.AddGameWideSystemMsgToChat(
-				" Something doesn't feel right? **BBBBBBBBBBBBBOOOOOOOOOOOOOOOOOOOMMMMMMMMMMMEMMEEEEE** ");
+				$"Anomoly Detected.. {gameObject.ExpensiveName()} has attempted to store itself within itself".Color(Color.red));
```

Comment:
dawwwwwwwwwwww

Reply:
if it ever shows up again, people are still going to be confused about it.
It's more understandable like this

Reply:
previous text made no sense